### DISCUSSION TITLE
Retire https://sourceforge.net/projects/elexis/ and aspects concerning this (Mailing-List, ...)

### DISCUSSION THREAD

**Discussion by col-panic** [elexis/elexis.github.io#2] [Q&A]
In https://sourceforge.net/p/elexis/mailman/elexis-develop/?viewmonth=202408&viewday=17 the discussion was started, whether to retire the Elexis sourceforge project, and migrate all open pointers and services to github.

Sourceforge has not been really used for quite a while - but the mailinglist seems alive and kicking. We would value everybodys opionen and practical tests, on whether THIS discussion area could become a replacement for the mailinglist.

Please try to give answers here! What is your pros and cons? Is there a technical solution to this? 

1. Can you opt-in for notifications to Discussions in here, like you are used with the MailingList?
2. ...

**Comment by marlovitsh** [comment]
Trying to answer
Harry

**Comment by rgwch** [comment]
And is there any non-german-speaking member? If no, we might switch to german here...

**Reply by debenol1** [reply]
Gute Idee!
Olivier

**Comment by rprch** [comment]
1. Ich finde es gut, wenn wieder alles an einer Stelle ist. Dabei wäre es auch gut, wenn mal die Handbuchseiten für die Elexis-User nutzbar sind. 
2. Ich setze den Vorschlag von Gerry um und antworte in DE :-)

**Reply by kappisj** [reply]
@debenol1 ist dabei das ehemalige Elexis-Wiki auch auf github zu übernehmen. Workflow-basierte Anleitungen haben wir neuerdings auch unter https://support.medelexis.ch/ aufgeschaltet (wobei die für eingefleischte Elexis-Hasen nichts Neues sein sollten ;-))

**Reply by debenol1** [reply]
Bin dabei, die verschiedenen Webressourcen zu Elexis zu sichten, zusammenzutragen und zu vereintheitlichen. Ich werde zeitnah eine Übersicht dazu posten

**Comment by rprch** [comment]
Danke.
Es wäre aber auch gut, wenn die Zugriffsrechte auf die Unterseiten "https://wikijs.medelexis.ch/" gehen.
Niklaus Giger hat 2022 in der Release-Note-3.9.0 auf dortige Unterseiten verwiesen, es ging bis Oktober 23 bis der Zugriff auf "https://wikijs.medelexis.ch/de/KonfigurationDokumentenManagement" ging, der Link "https://wikijs.medelexis.ch/de/KonfigurationQR-Rechnungen" ging damals - heute ist es umgekehrt.

**Comment by ngiger** [comment]
Am Sonntag, 18. August 2024, 19:59:22 CEST schrieb Marco Descher:
> In
> https://sourceforge.net/p/elexis/mailman/elexis-develop/?viewmonth=202408&v
> iewday=17 the discussion was started, whether to retire the Elexis
> sourceforge project, and migrate all open pointers and services to github.
I agree completely. Nice to see that more changes are coming for stuff I never 
had the time and energy to implement.
> 
> Sourceforge has not been really used for quite a while - but the mailinglist
> seems alive and kicking. We would value everybodys opionen and practical
> tests, on whether THIS discussion area could become a replacement for the
> mailinglist.
> 
> Please try to give answers here! What is your pros and cons? Is there a
> technical solution to this?
> 
> 1. Can you opt-in for notifications to Discussions in here, like you are
> used with the MailingList? 2. ...

* Moving to github discussions is okay as long as ALL (at least where 
possible) pointers to sourceforce.net/elexis are updated.
* Discussions should be held in english as Victor Mboka and myself are 
planning to revive the french version and we will have problems convincing 
possible supporters in the congo to learn first german to participate in 
discussions.

**Reply by col-panic** [reply]
@ngiger nice that you're back on Elexis! I followed your impressive bike journey through Europe!

I think that it should be perfectly okay, to have both german and english speaking topics.
I don't really mind doing it all in english - but maybe we have audiences which are not comfortable
in writing in English, and its fair to say that there are many more german speaking users of
Elexis in the moment than french speaking users.

If we communicate this right, maybe add a disclaimer to the top of the discussions concering
the language, than we should be good with both, no?
